src/components/TeacherComponents/hooks/downloadAndShareHooks/useResourceFormSubmit.test.ts

```diff
@@ -128,6 +128,8 @@ describe("useResourceFormSubmit", () => {
         "lesson",
         ["intro-quiz-questions"],
         true,
```

Comment:
We should maybe refactor this to take an object, so it's clear that the values are.